compiler/testData/codegen/bytecodeText/forLoop/stepped/emptyUntilProgressionToMinValue.kt

```diff
@@ -15,12 +15,11 @@ fun box(): String {
 //   // Standard form of loop over progression
 //   var inductionVar = 2
 //   val last = getProgressionLastElement(2, Int.MIN_VALUE - 1, 2)   // `(Int.MIN_VALUE - 1)` underflows to Int.MAX_VALUE
-//   val step = 2
 //   if (false && inductionVar <= last) {   // `false` comes from constant folding of `Int.MIN_VALUE != Int.MIN_VALUE`
 //     // Loop is not empty
 //     do {
 //       val i = inductionVar
-//       inductionVar += step
+//       inductionVar += 2
```

Comment:
Do I understand correctly that you're only fixing test data here? I don't see any changes in this test (and `illegalStepConst.kt`) neither in the IR, nor in the bytecode.

Reply:
Yes, I just took the opportunity to clean up the comments in the bytecode tests to reflect how the code actually looks like.

Reply:
I guess I should do the same in the other test files just to be consistent.